### ISSUE TITLE
Installation from .deb package on Debian 10 fails

### ISSUE THREAD

**Issue by amotl** [daq-tools/kotori#19]
A user of Kotori just reported that running Kotori on Debian fails 10. It gives:
```
root@hives1:/home/georges# tail -F /var/log/kotori/kotori.log
    from .compat import unicode
  File "/opt/kotori/lib/python2.7/site-packages/twisted/python/compat.py", line 611, in <module>
    import cookielib
  File "/usr/lib/python2.7/cookielib.py", line 32, in <module>
    import re, urlparse, copy, time, urllib
  File "/usr/lib/python2.7/copy.py", line 52, in <module>
    import weakref
  File "/usr/lib/python2.7/weakref.py", line 14, in <module>
    from _weakref import (
ImportError: cannot import name _remove_dead_weakref
```

**Comment by amotl** [comment]
Sorry that we didn't build appropriate packages for Debian 10 yet. Following the instructions outlined within #7 might help in the meanwhile.

**Comment by amotl** [comment]
Kotori 0.23.0 packages are now available for Debian "buster" 10.

The package repository is at [1]:
```
deb https://packages.elmyra.de/elmyra/foss/debian/ buster main foundation
```

[1] https://getkotori.org/docs/setup/debian-quickstart.html#register-with-package-repository

**Comment by amotl** [comment]
We've improved the packaging process and just released Kotori 0.24.5. It should work out of the box for Debian "buster" 10 now.
